### PR TITLE
chore: update README messaging

### PR DESCRIPTION
**Pull request by ssheng**
## Before submitting:

- [ ] Does the Pull Request follow [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary) naming? Here are [GitHub's
guide](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) on how to create a pull request.
- [ ] Does the code follow BentoML's code style, both `make format` and `make lint` script have passed ([instructions](https://github.com/bentoml/BentoML/blob/main/DEVELOPMENT.md#style-check-auto-formatting-type-checking))?
- [ ] Did you read through [contribution guidelines](https://github.com/bentoml/BentoML/blob/main/CONTRIBUTING.md#ways-to-contribute) and follow [development guidelines](https://github.com/bentoml/BentoML/blob/main/DEVELOPMENT.md#start-developing)?
- [ ] Did your changes require updates to the documentation? Have you updated
  those accordingly? Here are [documentation guidelines](https://github.com/bentoml/BentoML/tree/main/docs) and [tips on writting docs](https://github.com/bentoml/BentoML/tree/main/docs#writing-documentation).
- [ ] Did you write tests to cover your changes?

## Who can help review?

Feel free to tag members/contributors who can help review your PR.

### CODE REVIEW

**Review by jjmachan — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -18,26 +18,27 @@ things MLOps.
...
 - [🚀 bentoctl](https://github.com/bentoml/bentoctl): Fast model deployment on any cloud platform
 
 🐍 Python-first, scales with powerful optimizations
-- Parallelize compute-intense model inference workloads to scale separately from the serving logic
+- Parallelizes compute-intense model inference workloads to scale separately from the serving logic
 - Adaptive batching dynamically groups inference requests for optimal performance
-- Orchestrate distributed inference graph with multiple models via Yatai on Kubernetes
-- Easily configure CUDA dependencies for running inference with GPU
+- Orchestrates distributed inference graph with multiple models via Yatai on Kubernetes
+- Easily configures CUDA dependencies for running inference with GPU
```
Comment (sauyon) [inline README.md:41]:
```suggestion
- Easily configure CUDA dependencies for running inference with GPU
```

I think the user is doing the configuring here. It does feel out of place like this, though.

Comment (sauyon) [reply README.md:41]: Actually, how about:

```suggestion
- Allows for easy configuration of CUDA dependencies for running inference with GPU
```

File: README.md [hunk 2]
```diff
@@ -18,26 +18,27 @@ things MLOps.
 ## Why BentoML? ##
 
 🍱 Easily go from training to model serving in production
-- Support multiple ML frameworks natively: Tensorflow, PyTorch, XGBoost, Scikit-Learn and many more!
-- Define custom serving pipeline with pre-processing, post-processing and ensemble models
+- Supports multiple ML frameworks natively: Tensorflow, PyTorch, XGBoost, Scikit-Learn and many more!
+- Defines custom serving pipeline with pre-processing, post-processing and ensemble models
```
Comment (sauyon) [inline README.md:22]:
```suggestion
- Supports easy definition of full custom serving pipelines with pre-processing, post-processing and ensemble models
```

Comment (aarnphm) [reply README.md:22]: I would be like
```suggestion
- Enables custom serving pipelines with pre-processing, post-processing and ensemble models
```

File: README.md (hunk 1 above)
Comment (aarnphm) [reply README.md:41]: How about
```suggestion
- Enables CUDA dependencies customization for running inference with GPU with ease!
```

Comment (sauyon) [reply README.md:41]: I think that wordings a bit strange because it implies that users weren't able to do this before :P

Comment (aarnphm) [reply README.md:41]: which is true right 😆

File: README.md [hunk 3]
```diff
@@ -18,26 +18,27 @@ things MLOps.
...
+- Enables multi-model inference graph with model runners architecture
```
Comment (parano) [inline README.md:23]: lgtm 👍

File: README.md [hunk 4]
```diff
@@ -18,26 +18,27 @@ things MLOps.
...
 - [🦄️ Yatai](https://github.com/bentoml/yatai): Model Deployment at scale on Kubernetes
 - [🚀 bentoctl](https://github.com/bentoml/bentoctl): Fast model deployment on any cloud platform
 
 🐍 Python-first, scales with powerful optimizations
-- Parallelize compute-intense model inference workloads to scale separately from the serving logic
+- Parallelizes compute-intense model inference workloads to scale separately from the serving logic
 - Adaptive batching dynamically groups inference requests for optimal performance
-- Orchestrate distributed inference graph with multiple models via Yatai on Kubernetes
-- Easily configure CUDA dependencies for running inference with GPU
+- Orchestrates distributed inference graph with multiple models via Yatai on Kubernetes
```
Comment (parano) [inline README.md:40]:
```suggestion
- Orchestrates distributed inference graph with multiple models via [Yatai on Kubernetes](https://github.com/bentoml/Yatai)
```

**Pull request merged**